common-tools/clas-detector/src/main/java/org/jlab/detector/decode/CLASDecoder4.java

```diff
@@ -767,6 +767,7 @@ public static void main(String[] args){
         parser.addOption("-s", "0.5","solenoid current in the header bank");
         parser.addOption("-x", null,"CCDB timestamp (MM/DD/YYYY-HH:MM:SS)");
         parser.addOption("-v","default","CCDB variation");
+        parser.addOption("-skip","0", "number of events to skip");
```

Comment:
I think the parser doesn't support using strings that contain one of the other strings (in this case -skip contains -s). Did you check -s still works?

Reply:
will do ... meanwhile, note it's based on a Map<String,String> for option name->value (and the usual containsKey/get for access)